README.md

```diff
@@ -1,2 +1,30 @@
-# p4runtime
-Specification documents for the P4Runtime control-plane API
+# P4Rruntime Specification
+This directory contains protobuf files, specifications and related artifacts for all versions of the P4Runtime API. A separate directory is assigned to each major version, e.g. v1. Therein, tags are used to mark minor and patch release versions.
```

Comment:
Can we wrap the lines at 80 characters in all the files?

Reply:
I can't convert the whole file to 80-character wide. For  normal "prose" I can do this manually I suppose, but I can't do it to the tables, because Madoko is very fussy about making you put all content for a given row in one line. Using \ as a line-continuation does not seem to work. Perhaps someone knows better than I. The only alternative I am aware of in theses cases is to express the table as HTML, which sort of defeats the purpose. In fact I hand-converted all the HTML output following auto-conversion into markdown tables. Going forward we should consider the efficacy of Madoko tables when  constructing some of our content. Perhaps using native LaTex is the way to go in certain cases.

Reply:
No worries. I can merge this and I will take care of the 80-character wrap manually as I read the spec. Can you do it now for the non-Madoko files?

Reply:
Antonin, I've already done it and am reviewing prior to a new commit. Which non-Madoko files are you thinking of, the READMEs?

Reply:
Yes the READMEs
I'll review your commit as soon as it is pushed